src/customHooks/index.js

```diff
@@ -0,0 +1,4 @@
+import { useAxios } from "./useAxios";
+import { useDoc } from "./useDoc";
```

Comment:
suggestion: 
rename `useDoc` to `useDocumentTitle`
The Naming of the hook should explain what the hook does.